src/Swarm/TUI/View.hs

```diff
@@ -465,43 +466,56 @@ robotsListWidget s = hCenter table
   debugging = creative && cheat
   g = s ^. gameState
 
-helpWidget :: Widget Name
-helpWidget = helpKeys <+> helpCommands
+helpWidget :: Seed -> Maybe Port -> Widget Name
+helpWidget theSeed mport =
+  padTop (Pad 1) $
+    hBox . map (padLeftRight 2) $
+      [helpKeys, tips <=> padTop (Pad 1) info]
  where
+  tips =
+    vBox
+      [ txt "Have questions? Want some tips? Check out:"
+      , txt " "
+      , txt "  - The Swarm wiki, https://github.com/swarm-game/swarm/wiki"
+      , txt "  - The #swarm IRC channel on Libera.Chat"
+      ]
+  info =
+    vBox
+      [ txt ("Current seed: " <> into @Text (show theSeed))
+      , txt $ case mport of
+          Nothing -> "The web server is not running."
+          Just port -> "Web server running on port " <> into @Text (show port)
```

Comment:
This looks odd under the simple "Current seed: 0" - why not "Open port: 8080" or "Open port: none"?

Reply:
Sure, makes sense.  But I decided to put "web server port" instead of "open port" just to be clear what kind of port we're talking about, since new users won't have any idea.